src/v3/src/components/OpenOktaVerifyFPButton/OpenOktaVerifyFPButton.tsx

```diff
@@ -53,6 +54,9 @@ const OpenOktaVerifyFPButton: UISchemaElementComponent<{
   } = uischema;
   const [key, setKey] = useState<number>(0);
   const label = getTranslation(translations, 'label');
+  const { userIdentifier } = useWidgetContext();
+  const loginHintQueryParam = userIdentifier ? { login_hint: userIdentifier } : undefined;
```

Comment:
nit: consider `null`
```suggestion
  const loginHintQueryParam = userIdentifier ? { login_hint: userIdentifier } : null;
```